### PR TITLE
Bug 61805 - Allow simple HTTP request

### PR DESCRIPTION
**Pull request by orimarko**
## Description
Add simple HTTP request template
Creates an HTTP Request with Header Manager and View Results Tree
## Motivation and Context
Simplify creating a simple HTTP request
## How Has This Been Tested?
Create simple HTTP request with and without defaults on windows GUI
## Screenshots (if appropriate):
NA
## Types of changes

- New feature (non-breaking change which adds functionality)

## Checklist:

- [X ] My code follows the [code style][style-guide] of this project.
- [X ] I have updated the documentation accordingly.

[style-guide]: https://wiki.apache.org/jmeter/CodeStyleGuidelines

### CODE REVIEW

File: src/core/src/main/java/org/apache/jmeter/testelement/TestElement.java [hunk 1]
```diff
@@ -315,9 +315,9 @@
     String getName();
 
     /**
-     * Returns name of this element.
+     * Associates a name with this element.
```
Comment (FSchumacher) [inline src/core/src/main/java/org/apache/jmeter/testelement/TestElement.java:318]: This seems to be left over from some other PR?

File: bin/templates/templates.xml [hunk 2]
```diff
@@ -251,4 +251,26 @@
...
+        <name>Simple HTTP request</name>
+        <fileName>/bin/templates/simple-http-request-test-plan.jmx</fileName>
+        <description><![CDATA[
+        <h1>Template for simple HTTP request</h1>
+        <h2>JMeter Configuration</h2>
+        <p>Configure your HTTP request, set:
+            <ul>
+                <li>Full URL</li>
+                <li>Content Type</li>
+				<li>HTTP method</li>
```
Comment (FSchumacher) [inline bin/templates/templates.xml:264]: Nobody will see it, but could you try to keep the spacepolice happy?

File: bin/templates/templates.xml [hunk 3]
```diff
@@ -251,4 +251,26 @@
...
+				<li>Body data</li>
+            </ul>
+        </p>
+        ]]></description>
+        <parameters>
+            <parameter defaultValue="https://jmeter.apache.org/" key="url"/>
```
Comment (FSchumacher) [inline bin/templates/templates.xml:270]: Maybe we should take another domain than our own, so no DOS will result ;) In the documentation we use something ending on `example.invalid`. That way no one innocent will be harmed by the default settings.

File: bin/templates/templates.xml [hunk 4]
```diff
@@ -251,4 +251,26 @@
...
+            <parameter defaultValue="application/json" key="contentType"/>
+            <parameter defaultValue="POST" key="method"/>
+	    <parameter defaultValue="" key="body"/>
```
Comment (FSchumacher) [inline bin/templates/templates.xml:273]: We need space ;)

File: bin/templates/templates.xml (hunk 2 above)
Comment (orimarko) [reply bin/templates/templates.xml:264]: @FSchumacher I don't see an issue with spaves

**Comment by FSchumacher** [comment]
Thanks for the PR. I fixed the last spacing issues and squashed the commits.

**Pull request closed without merging**